testsuite/dockerfiles/server-all-in-one/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.suse.com/bci/bci-init:15.4
+FROM registry.suse.com/bci/bci-init:15.5
```

Comment:
not sure why here we are  using a SLE image instead openSUSE Leap. I know that the `bci-init` is based on tumbleweed, not on Leap, but using SLE as base and adding Leap  repos sounds  weird  to  me